cmd/monaco/supportarchive/supportarchive.go

```diff
@@ -56,6 +57,9 @@ func Write(fs afero.Fs) error {
 		ffState,
 		log.MemStatFilePath(),
 	}
+	if reportFilename, ok := os.LookupEnv(environment.DeploymentReportFilename); ok && len(reportFilename) > 0 {
```

Comment:
The method `os.LookupEnv` returns true if more than one environment variable is found with this name. So, I don't see a need in `len(reportFilename)`. Correct me if I'm wrong.

Reply:
The environment variable can be set but still be empty string
>  If the variable is present in the environment the value (which may be empty) is returned and the boolean is true

Reply:
I would vote for using `Getenv` as we don't have any error-case in thise case